### ISSUE TITLE
COVID Tracking Project Ending

### ISSUE THREAD

**Issue by zingbretsen** [CSSEGISandData/COVID-19#3690]
Is any data currently being source from the COVID Tracking Project? They are stopping their work aggregating data: [https://covidtracking.com/analysis-updates/covid-tracking-project-end-march-7](https://covidtracking.com/analysis-updates/covid-tracking-project-end-march-7)

Hospitalizations previously were sourced from them, before they were removed from this dataset. The README still states that CTP is used under the `Aggregated data sources` section:

> COVID Tracking Project: https://covidtracking.com/data. (US Testing and Hospitalization Data. We use the maximum reported value from "Currently" and "Cumulative" Hospitalized for our hospitalization number reported for each state.)

Is CTP still used as a source for US aggregated testing data? If so, will you adjust your source for testing?

**Comment by CSSEGISandData** [comment]
Yes, we are aware of this issue, and will come up with a solution soon. Thanks for your message.

**Comment by wellsangels** [comment]
Can you share any more about current plans to replace these data from Covid Tracking Project?
